packages/rs-drive-proof-verifier/src/error.rs

```diff
@@ -82,6 +82,37 @@ pub enum Error {
     /// Context provider error
     #[error("context provider error: {0}")]
     ContextProviderError(#[from] ContextProviderError),
+
+    /// Proof is stale; try another server
+    #[error(transparent)]
+    StaleProof(#[from] StaleProofError),
+}
+
+/// Received proof is stale; try another server
+#[derive(Debug, thiserror::Error)]
+pub enum StaleProofError {
```

Comment:
Are you sure we want to do that on the verifier side? Shouldn't be this logic located in SDK so we can guarantee freshness for all types of queries that have metadata?

Reply:
renamed to StaleNodeError and moved to Sdk.